domain/lib/src/model/biometric_authentication/biometric_authen_timeout.dart

```diff
@@ -29,15 +29,15 @@
 //  3 and <http://www.linshare.org/licenses/LinShare-License_AfferoGPL-v3.pdf> for
 //  the Additional Terms applicable to LinShare software.
 
-import 'biometric_authen_timeout_type.dart';
 import 'package:equatable/equatable.dart';
 
 class BiometricAuthenticationTimeout extends Equatable {
-  final int duration;
-  final BiometricAuthenTimeoutType timeoutType;
+  final int hour;
+  final int minute;
+  final int second;
```

Comment:
It uses for presentation?

Reply:
- This will be clear for others instead of only passing a millisecond parameter. So, need to separate into hour, minute, second
- Duration is a complex object with excess attributes. Timeout should be a pure class here with simple attributes